presto-docs/src/main/sphinx/connector/memory.rst

```diff
@@ -0,0 +1,67 @@
+================
+Memory Connector
+================
+
+The Memory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
+    to use it in a production environment. The intended use of this
+    connector at this point is purely for testing.
+
+.. warning::
+
+    After ``DROP TABLE`` memory is not released immediately. It is released
+    after next write access to memory connector.
```

Comment:
Isn't the `drop table` operation executed on a coordinator only? I don't know if there is some other way to communicate with workers beside `tableHandle` used in `select`, `insert` and `create table`. That's why memory is released on next `insert` or `create` statetment after `drop table`.